.jenkins/pytorch/test.sh

```diff
@@ -401,9 +403,18 @@ else
   test_benchmarks
   test_rpc
   if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
+    # code coverage for python sources
     pushd test
-    echo "Generating XML coverage report"
+    echo "Generating XML coverage report for python sources"
     time python -mcoverage xml
     popd
+
+    # code coverage for c++ sources
+    pushd build
```

Comment:
This is causing your `lcov ... build/` to fail since you are in the build directory when you try to run it

Reply:
Fixed